src/fandom-wirewax.plugin.js

```diff
@@ -0,0 +1,257 @@
+// fandom-jw-plugin.js
+
+console.log("FandomWirewaxPlugin loaded");
```

Comment:
Debugging?

Reply:
From the few modules I've looked at this seems like fairly standard practice

Reply:
Please, no, we have enough in console already; can it be at least debug?